config/initializers/doorkeeper.rb

```diff
@@ -0,0 +1,494 @@
+# frozen_string_literal: true
+
+Doorkeeper.configure do
+  # Change the ORM that doorkeeper will use (requires ORM extensions installed).
+  # Check the list of supported ORMs here: https://github.com/doorkeeper-gem/doorkeeper#orms
+  orm :active_record
+
+  # This block will be called to check whether the resource owner is authenticated or not.
+  resource_owner_authenticator do
+    # raise "Please configure doorkeeper resource_owner_authenticator block located in #{__FILE__}"
+    # Put your resource owner authentication logic here.
+    # Example implementation:
+    #   User.find_by(id: session[:user_id]) || redirect_to(new_user_session_url)
+    
+    user = User.last
```

Comment:
Terrible hack! I'm not familiar enough with rubygems.org and its use of Clearance to find the currently authenticated user.

See https://doorkeeper.gitbook.io/guides/ruby-on-rails/configuration#resource-owner-authentication